tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
     "compilerOptions": {
-      "paths": {
-        "*": ["types/*"]
-      },
+    "paths": {
```

Comment:
@m0cah this is correct, but you need to also make the changes I mention in my review from last week please

Reply:
Got it, will do it by tonight